integration/build.gradle

```diff
@@ -1 +1,36 @@
-// TODO! Maybe get started by copy/pasting from data-catalog?
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+plugins {
+    id 'io.spring.dependency-management'
+    id 'bio.terra.test-runner-plugin'
+}
+
+dependencyManagement {
+    imports {
+        mavenBom(SpringBootPlugin.BOM_COORDINATES)
+    }
+}
+
+// In `terra-data-catalog` the only `repositories { }` stanzas are in
+// the top-level `buildSrc` directory. But for some reason it does not get
+// inherited (or whatever the mechanism is) in this project.
+repositories {
+    maven {
+        url 'https://broadinstitute.jfrog.io/artifactory/plugins-snapshot'
+    }
+    gradlePluginPortal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.slf4j:slf4j-api'
+    implementation 'org.glassfish.jersey.inject:jersey-hk2'
+    implementation 'org.junit.jupiter:junit-jupiter-api'
+    implementation 'org.hamcrest:hamcrest'
+
+    // Google Dependencies
+    implementation 'com.google.auth:google-auth-library-oauth2-http:1.4.0'
+
+    // Terra Test Runner Library
+    implementation 'bio.terra:terra-test-runner:0.1.4-SNAPSHOT'
```

Comment:
You should also update the dependency for this in https://github.com/DataBiosphere/terra-batch-analysis/blob/main/buildSrc/build.gradle#L22 so it uses the same version

Reply:
There is most likely a way to avoid the duplicate dependency but I don't know how to do it. This is case where plugin dependencies and library dependencies use different configurations, which is usually what you want.